client/src/components/pages/home/case-study.tsx

```diff
@@ -1,7 +1,6 @@
-import { useIntl, FormattedMessage } from 'react-intl';
-import myImage from '/src/assets/images/case-study-placeholder.png';
 import caseStudyStyles from "./case-study.module.css";
```

Comment:
```suggestion
import { useIntl, FormattedMessage } from 'react-intl';
import caseStudyStyles from "./case-study.module.css";
```
Hmm, It seems my "quick fix" change actually broke this in a different way 😅